arch/xtensa/core/xtensa-asm2.c

```diff
@@ -163,17 +163,6 @@ void z_xtensa_dump_stack(const z_arch_esf_t *stack)
 #endif
 
 	LOG_ERR(" ** SAR %p", (void *)bsa->sar);
-
-#ifdef CONFIG_XTENSA_MMU
-	uint32_t vaddrstatus, vaddr0, vaddr1;
-
-	__asm__ volatile("rsr.vaddrstatus %0" : "=r"(vaddrstatus));
-	__asm__ volatile("rsr.vaddr0 %0" : "=r"(vaddr0));
-	__asm__ volatile("rsr.vaddr1 %0" : "=r"(vaddr1));
```

Comment:
These actually is neat because you can see which address is at fault, especially fault on data as EXCVADDR points to the code but not the data address.

Reply:
> EXCVADDR points to the code but not the data address.

This is not true. EXCVADDR points to the faulting data address in most of the exceptions, epc1 points to the faulting instruction.
vaddrstatus/vaddr* were introduced in the very recent MMU version that can work with cores having more that one load/store unit, MMUv3 (the one in dc233c) doesn't have it.

Reply:
Weird... maybe there is something wrong with the xt-ocd and GDB I have been using on the hardware. Will need to double check.